demos/transform/lv_demo_transform.c

```diff
@@ -122,7 +122,7 @@ static lv_obj_t * card_create(void)
     lv_obj_t * name = lv_label_create(card);
     lv_label_set_text(name, "Pavel Svoboda");
     lv_obj_set_grid_cell(name, LV_GRID_ALIGN_START, 2, 1, LV_GRID_ALIGN_CENTER, 0, 1);
-    lv_obj_set_style_text_font(name, &lv_font_montserrat_18, 0);
+    lv_obj_set_style_text_font(name, LV_FONT_MONTSERRAT_18 ? &lv_font_montserrat_18 : LV_FONT_DEFAULT, 0);
```

Comment:
To be sure that the benchmark is the same for everyone I would rather add this at beginning of the file.

```
#if LV_FONT_MONTSERRAT_14 == 0
#error "LV_FONT_MONTSERRAT_14 is required for lv_demo_benchmark. Enable it in lv_conf.h."
#endif 

#if LV_FONT_MONTSERRAT_18 == 0
#error "LV_FONT_MONTSERRAT_18 is required for lv_demo_benchmark. Enable it in lv_conf.h."
#endif 

#if LV_FONT_MONTSERRAT_24 == 0
#error "LV_FONT_MONTSERRAT_24 is required for lv_demo_benchmark. Enable it in lv_conf.h."
#endif 
```

Reply:
If benchmark is listed as a demo, should the font not be restricted?

Reply:
I think for demo it's normal to use more than on font. The widget's demo also require some non default fonts.